src/BindPattern.jl

```diff
@@ -500,6 +531,16 @@ function shred_where_clause(
     end
 end
 
+#
+# getvars
+#
+# get all symbols in an expression
+#
+getvars(e)         = Set{Symbol}()
+getvars(e::Symbol) = startswith(string(e), '@') ? Set{Symbol}() : push!(Set{Symbol}(), e)
```

Comment:
why switch from the postwalk to this version? is it as exhaustive? more exhaustive?

Reply:
Because the postwalk version rewrote the tree and threw away the result.  This version is much faster, as it just scans.